firefox/Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN apt-get update && apt-get install -y \
 	firefox \
 	hicolor-icon-theme \
 	libasound2 \
+    ffmpeg \
```

Comment:
can you make it a tab so sorry and alphabetical order :) thanks!

Reply:
no problem, updated :+1: